wordpress-mobile/WordPress-iOS
# Configure Kanvas aspect ratio cropping

Kanvas PR: https://github.com/tumblr/kanvas-ios/pull/107

Adds 9:16 aspect ratio cropping + UI for the Stories editor.

<a href="https://user-images.githubusercontent.com/3250/113606196-e8850780-9604-11eb-80dd-579b2dba8ff5.PNG"><img src="https://user-images.githubusercontent.com/3250/113606196-e8850780-9604-11eb-80dd-579b2dba8ff5.PNG" width="300"></a>

## Testing

1. Create a new Story
2. Take a Photo/Video or select from Media Library
3. Ensure that the media is shown in 9:16 aspect ratio while editing. This varies on device but should include borders when the device screen is too large (iPhone 11, iPhone 11 Pro, etc.) and no borders on devices which are already 9:16 (iPhone 8). [This list](https://ios-resolution.com) provides the logical width/height of the device and 9:16 ratios can be calculated from there.
4. Publish the story
5. Ensure that published media is exported in 9:16 aspect ratio.

## Regression Notes

1. Potential unintended areas of impact
The Stories editor.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested the Stories editor.

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
